globals.go

```diff
@@ -870,6 +870,19 @@ func (k SenderKey) String() string {
 	}
 }
 
+// DevProdJiraServiceField defines a required field for DEVPROD tickets, which we sometimes auto-generate.
+// Using "Other" prevents this from getting out of sync with service naming too quickly.
+var DevProdJiraServiceField = map[string]string{
+	"id":    devProdServiceId,
+	"value": devProdServiceValue,
+}
+
+const (
+	DevProdServiceFieldName = "customfield_24158"
+	devProdServiceId        = "27020"
```

Comment:
Since these two are only used above, do we need to make them constants? The name does reveal more about it but if we use the values directly above, the name of the map also helps decipher what they are

Reply:
I think this is just a matter or preference but I think its a bit more correct to keep them as constants (and they're private so)